### ISSUE TITLE
accessibility : keyboard zoom out on laptop (missing keycode 54)

### ISSUE THREAD

**Issue by remilev** [Leaflet/Leaflet#3362]
I'm using leaflet stable 0.7.3 with an AZERTY laptop keyboard (french layout).
Hitting "+" (shift + "=") will zoom in as expected, but "-" (6 without shift) won't zoom out.
Issue tested on firefox 37 and chrome 41. Can be test on the map at http://leafletjs.com/

In the minified source, I see :
<code>o.Map.Keyboard=o.Handler.extend({keyCodes:{left:[37],right:[39],down:[40],up:[38],zoomIn:[187,107,61,171],zoomOut:[189,109,173]}</code>

By adding keycode 54 for zoomOut, it will fix this issue :
<code>o.Map.Keyboard=o.Handler.extend({keyCodes:{left:[37],right:[39],down:[40],up:[38],zoomIn:[187,107,61,171],zoomOut:[189,109,173,54]}</code>

Is this a real issue or just a problem with my keyboard ?

**Comment by Eschon** [comment]
I think this is a leaflet issue. I never really got the keyboard controls to work for me but since I din't really need them I didn't look into it. Seeing this issue brought me some ideas and I figured out that the actual + and - keys on my keyboard do nothing but the keys for ß (-) and ` (+) on my german QERTZ keyboard work for me.

**Comment by mourner** [comment]
Yeah, looks like an easy fix. I've never seen an AZERTY keyboard before and you're the first one to report.
